draft-ietf-git-using-github.md

```diff
@@ -256,6 +248,14 @@ editors are able to do in order to serve these goals.  Chairs SHOULD
 periodically consult with document editors to ensure that policies are effective
 and not unjustifiably constraining progress.
 
+A document editor can still use GitHub independently for documents that they
+edit, even if the Working Group does not expressly choose to use GitHub.  Any
+such public repository MUST follow the guidelines in BCPs 78 and 79
+({{!COPYRIGHT=RFC5378}}, {{!IPR=RFC8179}}) and bear notices to that effect.
```

Comment:
can we also replace these with the note well?

Reply:
I honestly don't know.

Reply:
In that case, can we require a link to the Note Well on top of what's already required?